test/check-application

```diff
@@ -1447,14 +1447,73 @@ class TestFiles(testlib.MachineCase):
         b.wait_in_text(".pf-v5-c-modal-box__title-text", "bin")
         b.wait_in_text("#edit-permissions-owner", "root")
         b.wait_in_text("#edit-permissions-group", "root")
-
-        # As normal user you cannot change user/group permissions
-        b.drop_superuser()
+        b.click("button.pf-m-link")
+        b.wait_not_present(".pf-v5-c-modal-box")
```

Comment:
Ugh, forgotten to close the dialog -- how did that work before, does dropping superuser magically close it? Anyway, moot now, thanks for fixing!